Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+ruff = "*"
```

Comment:
Is Ruff a runtime dependency?

Reply:
It is being demonstrated in session 4, so I would say yes.